source/includes/options-mongosqld.yaml

```diff
@@ -101,23 +101,57 @@ description: |
   ``admin`` and ``local`` databases.
 ---
 program: mongosqld
-name: sampleSource
+name: schemaSource
 directive: option
 args: <db-name>
 description: |
-  .. versionadded:: 2.3
+  .. versionchanged:: 2.11
+
+     Renamed ``--sampleSource`` to ``-schemaSource``     
 
-  Required when :option:`--sampleMode <mongosqld --sampleMode>` is set
-  to write.   Specifies a database to store schema information.
+  Required when :option:`--schemaMode <mongosqld --schemaMode>` is set
+  to ``auto``. Specifies a database to store schema information.
   
   .. note::
 
      If you do not specify either a schema file with the
      :option:`--schema` option or a database with the
-     ``--sampleSource`` option, :binary:`~bin.mongosqld` holds its
+     ``--schemaSource`` option, :binary:`~bin.mongosqld` holds its
      schema in memory.
 
   .. include:: /includes/sampling-ref-chart-link.rst
+---
+program: mongosqld
+name: schemaName
+directive: option
+args: <db-name>
+default: "``defaultSchema``"
+description: |
+  .. versionadded:: 2.11    
+
+  Optional. The name of the schema to load from or write to
+  the :option:`--schemaSource <mongosqld --schemaSource>` database
+  depending on the value of :option:`--schemaMode <mongosqld --schemaMode>`:
```

Comment:
It's been a while since I've worked with this program, but at this point I'm having a hard time figuring this sentence out. It sounds as if the schema in "The name of the schema to load from..." is a schema file, but I think it's actually a set of definitions stored in a collection, is that correct? I think we should avoid speaking of "the schema" in the abstract; we should refer either to a text file or a collection which can hold one or more schema definitions.

Reply:
The schema file is from mongodrdl and isn't covered here.

You could start up a BIC and have it sample and write the data:
```
mongosqld --schemaMode auto --schemaSource foo --schemaName bar
```

Let's say the sampling was good and things worked well, you could load that sampled data next time and have the BIC not sample or overwrite anything:
```
mongosqld --schemaMode custom --schemaSource foo --schemaName bar
```

I can't think of any other way around not using "schema" unless we go with "schema data" or "schema information"

Reply:
or "schema mapping"

Reply:
I'd go with "schema data".